### PR TITLE
Added 'cellMessage' property for cell view schema

### PR DESCRIPTION
**Pull request by trananh-khoa**
### This project uses [semver](semver.org), please check the scope of this pr:
 - [ ] #none# - documentation fixes and/or test additions
 - [ ] #patch# - backwards-compatible bug fix
 - [x] #minor# - adding functionality in a backwards-compatible manner
 - [ ] #major# - incompatible API change

# CHANGELOG
Added 'cellMessage' property for cell view schema

### CODE REVIEW

File: src/validator/view-schemas/v2.js [hunk 1]
```diff
@@ -113,6 +113,11 @@ const definitions = {
         '$ref': '#/definitions/arrayOptions'
       },
 
+      // Text to display at bottom of cell
+      cellMessage: {
```
Comment (EWhite613) [inline src/validator/view-schemas/v2.js]: Probably needs a better name more descriptive name. Is this only used for sections?

Comment (trananh-khoa) [reply src/validator/view-schemas/v2.js]: Yeah only for sections, would something like `sectionText`, `sectionMessage`, or `sectionFootnote` work?

Comment (trananh-khoa) [reply src/validator/view-schemas/v2.js]: Going with `sectionFootnote`

**Review by sophypal — COMMENTED** [review]
Only problem I have with `sectionFootnote` is that it's specific to a section while the cell definition can either be an ordinary `cell` or a `section`. Will a `cell` ever have the need for a footnote, and if it did, would you have multiple definitions like `sectionFootnote` and `cellFootnote`. The cell is either a `section` or not. There shouldn't be a use case where you'd want both.

**Comment by trananh-khoa** [comment]
@sophypal that makes sense, will rename it to just `footnote` to keep it flexible

**Review by sophypal — APPROVED** [review]

**Comment by sophypal** [comment]
👍

**Pull request merged**